native/src/main/java/org/wssec/Constants.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.wssec;
+
+public class Constants {
```

Comment:
Shall we introduce a private constructor?

Reply:
Is'nt it possible to define these constants within the classes where they are utilized?

Reply:
Fixed in [d5fc6c7](https://github.com/ballerina-platform/module-ballerina-soap/pull/103/commits/d5fc6c724e4c4e665e68067f131344ffcc9e9f21)